### PR TITLE
Issue 1580

### PR DESCRIPTION
**Pull request by aayushbhan**
Fixes for strongbox/strongbox/issues/1580.

Documentation for Gradle Tool Integration and Maven Tool Integration.

### CODE REVIEW

**Comment by aayushbhan** [comment]
https://github.com/strongbox/strongbox/issues/1580

**Comment by aayushbhan** [comment]
https://github.com/strongbox/strongbox/issues/1580

File: mkdocs.yml [hunk 1]
```diff
@@ -94,6 +94,8 @@ nav:
     - Tool Integration:
       - NPM Integration Example: user-guide/tool-integration/npm-example.md
       - SBT Integration Example: user-guide/tool-integration/sbt-example.md
+      - MAVEN Example: user-guide/tool-integration/maven-example.md
```
Comment (carlspring) [inline mkdocs.yml]: No need for these to be all uppercase. Please, change them to `Maven Example` and `Gradle Example`. Thanks!

File: wiki/_sidebar.md [hunk 2]
```diff
@@ -35,6 +35,8 @@
 * [[Tool Integration]]
   * [[NPM Integration Example]]
   * [[SBT Integration Example]]
+  * [[MAVEN Example]]
+  * [[GRADLE Example]]
```
Comment (carlspring) [inline wiki/_sidebar.md]: And here.

**Comment by carlspring** [comment]
I don't know, if you've actually tested how this looks when it's rendered, but the bullet-points are all broken for Gradle.

In pull requests checks, there's a link to `deploy/netlify` where you can see the rendered content when you push. (You should really be doing this locally, to begin with...). Could you please make sure that things look okay? Thanks!

**Comment by carlspring** [comment]
Please, rebase.

**Comment by carlspring** [comment]
I've fixed the conflicts. Thanks for your help and congratulations on your fist pull request! :)

**Pull request merged**